callback-translation/README.MD

```diff
@@ -0,0 +1,189 @@
+# Human validation of text translation via Webhooks callbacks
```

Comment:
I think they are referred as "callback endpoints" with our docs some maybe change "Webhooks callbacks" to "Workflows callback endpoints"

Reply:
You also want to link to this README from the main README of the repo

Reply:
Changed the nomenclature to callback endpoints, and updated README with the link